src/App.js

```diff
@@ -1,11 +1,22 @@
 import AppHeader from './components/AppHeader'
+import Card from './components/Card/Card'
+import Content from './components/Content/Content'
```

Comment:
I know it's probably because of auto-completion: 
```suggestion
import Content from './components/Content'
```